functions.py

```diff
@@ -1,36 +1,35 @@
 import time
+
+def ReadLines(nameoffile):
+    with open(nameoffile,'r') as f:
+        arrayline = f.readlines()
+        return arrayline
+
 #how much is not taxed
 def SellWithoutTax():
-    f = open('new_register.txt','r')
-    try:
-        arrayline = f.readlines()
-        sum = 0.0
-        i = 0
+    arrayline = ReadLines('new_register.txt')
+    sum = 0.0
+    i = 0
+    arr = arrayline[i].split(',', 2)
+    while (time.time() - float(arr[1])) >= 31536000.0:

```

Comment:
I would rather just iterate in a for loop with a break ... 

your code would fail if state file contain only times greater than a year. Pls try to find out why, if not I'll give you a hint.